acceptance/tests/options/config_file/no_custom_facts_and_custom_dir.rb

```diff
@@ -0,0 +1,43 @@
+# This test verifies that setting both custom-dir and no-custom-facts results in an error
+test_name "99994: config option no-custom-facts conflicts with custom-dir" do
+  tag 'risk:low'
+
+  require 'facter/acceptance/user_fact_utils'
+  extend Facter::Acceptance::UserFactUtils
+
+  content = <<EOM
+Facter.add('custom_fact') do
+  setcode do
+    "testvalue"
+  end
+end
+EOM
+
+  agents.each do |agent|
+    custom_dir = get_user_fact_dir(agent['platform'], on(agent, facter('kernelmajversion')).stdout.chomp.to_f)
```

Comment:
Why is this not a simple `agent.tmpdir('custom_dir')` call like the other tests?